app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= link_to "Sign Up", root_path %>
```

Comment:
Could this be link to 'Home' and then we have the same link on all pages? (although it's not in the user stories)

Reply:
Great idea but maybe once logged in - the 'home' page could be the posts page so that the user can always navigate there form wherever they are in the app.